index.js

```diff
@@ -96,6 +107,50 @@ function parse(str, options) {
  */
 
 function serialize(name, val, options) {
+  if( typeof name === 'object') {
+    var cookies = name;
+    var serializeOptions = val;
+
+    var cookieNames = Object.keys(cookies);
+    if(0 === cookieNames.length) {
+      return '';
+    } else if(cookieNames.length > 1) {
```

Comment:
Why the `else if` case here? There is some case there `Object.keys` would return negative numbers?